frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -92,6 +92,15 @@ export const BucketsItem: React.FC<BucketsItemProps> = React.memo(
   ),
 );
 
+export const getTitleComponent = (objectsCount: string, link: string) => ({ children }) => (
+  <div className="nb-buckets-card__buckets-status-title">
+    {link ? <Link to={link}>{children}</Link> : <div>{children}</div>}
+    <div className="co-dashboard-text--small text-secondary">
```

Comment:
I think this part should not be visible/or should show a some loading state, while the resource is loading, otherwise it shows `0 Objects`